src/Tmds.Systemd/Journal.cs

```diff
@@ -207,7 +207,11 @@ public static unsafe LogResult Log(LogFlags flags, JournalMessage message)
 
         private static void ErrorWhileLogging(string cause)
         {
-            Console.WriteLine($"Error writing message to journal: {cause}");
+            const string SD_ERR = "<3>";
+            var pri = Console.IsErrorRedirected && ServiceManager.IsRunningAsService 
+                ? SD_ERR 
+                : null;
+            Console.Error.WriteLine("{0}Error writing message to journal: {1}", pri, cause);
```

Comment:
Are you hitting the `ErrorWhileLogging`? I've put it in, just in case.

Reply:
Yes, I do, when the message is exceeded (errno=EMSGSIZE). What made me consider doing this PR is that I found out about this problem (and that I was losing logs due to logging too large messages, but that's a separate issue), only when browsing logs manually, since alerts that we have on any errors and on "many" warnings do not trigger on "info" severity level logs. I believe that other people log monitoring facilities are alike in testing for severity when considering anything for alerting.

Reply:
I spent like 20 minutes poking back and forth looking at our graylog instance (we've got a journald->graylog shipping setup with some filters, to collect logs centrally), adding more and more logging and wondering why some error messages just seem to drop, until I've got the idea that I should check other severities and/or locations =)